test/blackbox/metrics.bats

```diff
@@ -32,6 +32,7 @@ function setup_file() {
     zot_config_file=${BATS_FILE_TMPDIR}/zot_config.json
     zot_htpasswd_file=${BATS_FILE_TMPDIR}/zot_htpasswd
     htpasswd -Bbn ${AUTH_USER} ${AUTH_PASS} >> ${zot_htpasswd_file}
+    htpasswd -Bbn ${METRICS_USER} ${METRICS_PASS} >> ${zot_htpasswd_file}
```

Comment:
Make sure we add a test to cover no authn case also, meaning if authn is not configured, then anyone can still access.

Reply:
This scenario is covered in TestMetricsAuthentication in monitoring_test.go